### ISSUE TITLE
Escape issue.

### ISSUE THREAD

**Issue by miniminion100** [Nurrl/Dckuino.js#24]
```
Arduino: 1.8.2 (Windows Store 1.8.2.0) (Windows 10), Board: "Arduino Leonardo"

C:\Users\lsjon\OneDrive\Documents\Arduino\hotdog_backround\hotdog_backround.ino: In function 'void setup()':

hotdog_backround:62: error: 'KEY_LEFT_ESC' was not declared in this scope

   Keyboard.press(KEY_LEFT_ESC);

                  ^

C:\Users\lsjon\OneDrive\Documents\Arduino\hotdog_backround\hotdog_backround.ino: At global scope:

hotdog_backround:114: error: expected initializer at end of input

 void loop() 

           ^

exit status 1
'KEY_LEFT_ESC' was not declared in this scope

```

**Comment by Nurrl** [comment]
This issue is a duplicate of #20, and has been fixed with ac3353c.

**Comment by Nurrl** [comment]
Please recompile the code.
